coast/NEMO.py

```diff
@@ -302,7 +302,7 @@ def transect_indices(self, start: tuple, end: tuple) -> tuple:
         return jj1, ii1, line_length
     
     @staticmethod
-    def interpolate_in_space(model_array, new_lon, new_lat):
+    def interpolate_in_space(model_array, new_lon, new_lat, mask=None):
```

Comment:
This method looks like it might be more efficient than coast.subset_indices_lonlat_box